tests/pytorch_tests/model_tests/test_feature_models_runner.py

```diff
@@ -116,6 +116,7 @@
 from tests.pytorch_tests.model_tests.feature_models.tpc_test import TpcTest
 from tests.pytorch_tests.model_tests.feature_models.uniform_activation_test import \
     UniformActivationTest
+from model_compression_toolkit.target_platform_capabilities.constants import PYTORCH_KERNEL, BIAS
```

Comment:
I think that the changes in this file are unnecessary. If so, please remove them